### ISSUE TITLE
build = false is not respected

### ISSUE THREAD

**Issue by vrnvorona** [jonsamwell/flutter_gherkin#91]
My config
```final config = FlutterTestConfiguration()
    ..features = [Glob('test_driver/features/scenarios/**.feature')]
    ..reporters = [ProgressReporter(), TestRunSummaryReporter(), JsonReporter()]
    ..stepDefinitions = [...AuthSteps().steps] // from features/step_list.dart
    ..restartAppBetweenScenarios = true // default
    ..targetAppPath = 'test_driver/app.dart' // default
    ..exitAfterTestRun = false
    ..build = false
    ..logFlutterProcessOutput = true
  ;
```
I run as `dart test_driver/app_test.dart`

And in console (because of logFlutterProcessOutput param) I see
```
Launching test_driver/app.dart on iPhone 11 in debug mode...
Running Xcode build...                                          
Xcode build done.                                           39.7s
```

**Comment by jonsamwell** [comment]
That parameter is just forwarded to the flutter driver process so that is the actual thing deciding whether to build the app or not.  If you run flutter_gherkin using verbose logging you can see the command it sends to flutter driver.  I suspect something has changed in your code that invalidated the last build so the flutter driver process thinks it needs to rebuild

**Comment by vrnvorona** [comment]
I am running it without changes, but yeah, i just narrowed it down to flutter run too. Sorry for not verified issue

**Comment by vrnvorona** [comment]
Though, is there a way to not run flutter run but use session from flutter drive?

**Comment by jonsamwell** [comment]
Yes there is see
https://github.com/jonsamwell/flutter_gherkin#debugging-the-app-under-test

It's not too well documented now and the debugger session uri starts with a
ws:// not a http:// as the docs suggest. I need to update it.

**Comment by vrnvorona** [comment]
Optionally maybe implement --use-application-binary, --no-build for flutter run seems to have no effect https://github.com/flutter/flutter/issues/68529

**Comment by jonsamwell** [comment]
@vrnvorona you can connect to an existing app see here https://www.wfa.com.au/water-filters/

I will try and get the flag to stop the driver terminating the app under test in soon.

**Comment by jonsamwell** [comment]
Fixed in latest v1.1.9 release see https://github.com/jonsamwell/flutter_gherkin#keepapprunningaftertests
